server/src/ModelCompiler.ts

```diff
@@ -71,7 +74,13 @@ export class ModelCompiler {
           if (result.isErr()) {
             this.onCompilationErrorEmitter.fire(result.error);
           } else {
-            this.onCompilationFinishedEmitter.fire(result.value);
+            const value =
+              // For some reason rpc server don't compile intermediate models for packages, for this case we get results from target folder
+              result.value === ' ' && modelPath.includes('intermediate') && !modelPath.includes(path.sep)
```

Comment:
Seems that the problem is not in `intermediate`. Please, see installed packages configs:
**dbt-language-server/e2e/projects/project-with-packages:**
```
packages:
  - package: fivetran/github
    version: 0.4.0
```
_dbt_packages/github/dbt_project.yml:_
```
models:
  github:
    materialized: table
    intermediate:
      materialized: ephemeral
```

**My test project:**
```
packages:
  - package: fivetran/salesforce
    version: 0.5.1
```
_dbt_packages/salesforce/dbt_project.yml_
```
models:
  salesforce:
    materialized: table
    intermediate:
      materialized: ephemeral
```

Reply:
I don't know how different versions of rpc server will handle `ephemeral` models. Please consider obtaining compiled sql for all packages from files in `target`.

Reply:
Thanks for the great finds, I'll try to figure it out.

Reply:
Added fallback for all models with materialized == ephemeral.